soc/riscv/riscv-privilege/andes_v5/Kconfig.defconfig.ae350

```diff
@@ -18,4 +18,12 @@ config IDLE_STACK_SIZE
 config TEST_EXTRA_STACK_SIZE
 	default 1024
 
+config MP_NUM_CPUS
+	int "Number of CPUs/cores"
+	default 1
+	range 1 8
+	help
+	  Number of multiprocessing-capable cores available to the
+	  multicpu API and SMP features.
```

Comment:
CI complains about this option:
> Kconfig node 'MP_NUM_CPUS' found with prompt or help in soc/riscv/riscv-privilege/andes_v5/Kconfig.defconfig.ae350.

Reply:
Thanks for your remind.
According to the Kconfig document, the prompt or help can't defined multiple times.
I have fixed it.